newsroom/src/main/java/com/theguardian/newsroom/model/Event.kt

```diff
@@ -0,0 +1,5 @@
+package com.theguardian.newsroom.model
+
+import java.util.*
+
+class Event(val source: String, val title: String, val message: String, val timestamp: Date)
```

Comment:
Make this a `data class`

Reply:
Also, should we consider using that library [ThreeTen](https://github.com/JakeWharton/ThreeTenABP) for `Date` stuff in this project, as we're starting from a clean slate?

Reply:
@maxspencer I'm going to add ThreeTen when this is merged: https://github.com/guardian/Newsroom/pull/11

Quite a lot has changes so I'm trying to avoid conflicts.